src/cool_seq_tool/mappers/exon_genomic_coords.py

```diff
@@ -413,13 +413,12 @@ async def genomic_to_tx_segment(
         transcript: str | None = None,
         gene: str | None = None,
         coordinate_type: CoordinateType = CoordinateType.INTER_RESIDUE,
+        assembly: Assembly = Assembly.GRCH38,
```

Comment:
I wonder if we should rename this to `starting_assembly`, `initial_assembly` or something similar.